specification/vmware/resource-manager/Microsoft.AVS/preview/2020-07-17-preview/vmware.json

```diff
@@ -0,0 +1,4002 @@
+          "type": "string"
+        },
+        "provisioningState": {
+          "description": "The status of the Arc configuration",
+          "type": "string",
+          "readOnly": true,
+          "enum": [
+            "Succeeded",
+            "Failed",
+            "Cancelled",
+            "Running"
+          ],
+          "x-ms-enum": {
+            "name": "ArcConfigurationState",
+            "modelAsString": true
+          }
+        },
+        "infraResourceGroupName": {
+          "description": "Name of the Arc infrastructure resource group for a private cloud",
+          "type": "string",
+          "readOnly": true
+        },
+        "aksClusterId": {
+          "description": "ARM ID of the AKS cluster resource deployed in infra resource group",
+          "type": "string",
+          "readOnly": true
+        },
+        "arcZoneId": {
+          "description": "ARM ID of the ArcZone resource deployed in arc resource group",
+          "type": "string",
+          "readOnly": true
+        }
+      }
+    },
+    "GlobalReachConnectionList": {
+      "description": "A paged list of global reach connections",
+      "properties": {
+        "value": {
+          "description": "The items on a page",
+          "type": "array",
+          "readOnly": true,
+          "items": {
+            "$ref": "#/definitions/GlobalReachConnection"
+          }
+        },
+        "nextLink": {
+          "description": "URL to get the next page if any",
+          "type": "string",
+          "readOnly": true
+        }
+      }
+    },
+    "GlobalReachConnection": {
+      "description": "A global reach connection resource",
+      "allOf": [
+        {
+          "$ref": "#/definitions/Resource"
+        }
+      ],
+      "properties": {
+        "properties": {
+          "description": "The properties of a global reach connection resource",
+          "$ref": "#/definitions/GlobalReachConnectionProperties",
+          "x-ms-client-flatten": true
+        }
+      }
+    },
+    "GlobalReachConnectionProperties": {
+      "description": "The properties of a global reach connection",
+      "properties": {
+        "provisioningState": {
+          "description": "The state of the  ExpressRoute Circuit Authorization provisioning",
+          "type": "string",
+          "readOnly": true,
+          "enum": [
+            "Succeeded",
+            "Failed",
+            "Updating"
+          ],
+          "x-ms-enum": {
+            "name": "GlobalReachConnectionProvisioningState",
+            "modelAsString": true
+          }
+        },
+        "addressPrefix": {
+          "description": "The network used for global reach carved out from the original network block provided for the private cloud",
+          "type": "string",
+          "readOnly": true
+        },
+        "authorizationKey": {
```

Comment:
is this a secret?  If so please mark it with x-ms-secret and x-ms-mutability and ensure it doesn't get returned on responses

Reply:
We do not consider this a secret.   We have had comments in the past on this.  I'll dig up the conversation if I can find it in a past PR

Reply:
@jspearman3 ^^^